.github/workflows/rolling-instance-actions.yml

```diff
@@ -121,6 +121,17 @@ jobs:
         client-secret: ${{ secrets.CLIENT_SECRET_PPDEVTOOLS }}
         tenant-id: ${{ env.WF_TENANT_ID }}
 
+    - name: Test deploy-package action with username-password
+      uses: ./deploy-package
+      with:
+        environment-url: ${{ steps.create-source-environment.outputs.environment-url }}
+        user-name: ${{ env.WF_USERNAME }}
+        password-secret: ${{ secrets.PASSWORD_PPDEVTOOLS }}
+        tenant-id: ${{ env.WF_TENANT_ID }}
+        package: ''
```

Comment:
I am not entirely sure what should be the path to the package here for rolling-instance-actions.